radio/src/targets/horus/CMakeLists.txt

```diff
@@ -25,6 +25,9 @@ set(FLAVOUR horus)
 set(RAMBACKUP YES)
 set(LUA YES)
 set(PPM_LIMITS_SYMETRICAL YES)
+# for size report script
+set(CPU_TYPE_FULL STM32F429xI)
```

Comment:
It's a parameter/shortcut for the bash script to set the right memory segment sizes:  https://github.com/opentx/opentx/pull/4218/files#diff-8d137f69d1aafa5d21399516e3d9e0f6R76  
(The pin count doesn't matter and I didn't know it, hence the "x".)